### ISSUE TITLE
Conditional fields where pidValues come from checkboxes are not shown upon change

### ISSUE THREAD

**Issue by RafaPolit** [KETSE/casebox#13]
I have created a child node of a checkbox, and on its config I have the following JSON:
```
{
  "dependency": { 
    "pidValues": [1]
  }
}
```
If I select either yes or no on the checkbox, I never get the new field to appear.  If I save the case with the checkbox field set to 'yes' and the REOPEN the form to edit the case again, then the child field is visible as expected.

Is there any config that will allow me to show the child nodes 'on the fly' when the user selects 'yes' as the option for the parent checkbox?

Thanks in advance,
Rafa.

**Comment by tvitalie** [comment]
Hi Rafa, 
cant reproduce this issue in Chrome under windows.
Will create an ubuntu system as you'he described in /KETSE/casebox/issues/9 and will test on it.

Thank you for detailed descriptions on the issues you find.

**Comment by oburlaca** [comment]
@RafaPolit, are you using the mouse to select Yes/No?
If so, after you choose a value in the dropdown, you have to click Enter, or move to another field or click outside the checkbox field. This will "commit" your changes for the field, i.e. will set the value.

Using the keyboard works fine.

This can be added as a todo for UX improvement

**Comment by RafaPolit** [comment]
Yes, I am using the mouse.  Indeed, clicking outside the field 'commits' the changes, but yes, I believe it would be a nice improvement if that could happen upon selection.

Thanks,
Rafa.
